docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -438,6 +440,53 @@
       your Integrated Spark Master node's IP address.</para>
 
       <programlisting>https://192.168.56.101:8080</programlisting>
+
+      <sect2 id="Addl_Spark_Config">
+        <title>Integrated Spark Cluster Configuration Options</title>
+
+        <para>To customize your Integrated Spark cluster environment utilize
+        the provided <emphasis role="bold">spark-env.sh</emphasis> script.
+        Using the spark-env.sh script you can specify additional configuration
+        options for your Integrated Spark cluster.</para>
+
+        <programlisting>/etc/HPCCSystems/externals/spark/spark-env.sh </programlisting>
```

Comment:
This script won't exist by default, and the user will have to create it. Should we mention this?

Reply:
yes. will do.

Reply:
I think @Michael-Gardner might have made a change which automatically adds this script (at least as of 7.0.12) Mike can you confirm?

Reply:
@g-pan The above path changed from /etc/HPCCSystems/externals/spark/spark-env.sh to /etc/HPCCSystems/externals/spark-hadoop/spark-env.sh